reenact/templates/base.html

```diff
@@ -20,8 +20,9 @@
     <link rel="icon" href="{% static 'images/favicons/favicon.ico' %}" />
     {% block sources %}
     {% endblock sources %}
+    <link rel="stylesheet" href="{% static 'css/tailwind.css' %}" />
+    <link rel="stylesheet" href="{% static 'css/project.css' %}" />
```

Comment:
These two can stay within CSS.
You can use `{{block.super}}` in map.html in order to inherit them.